subprojects/docs/src/samples/build-organization/cross-project-output-sharing/kotlin/consumer/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+val sharedConfiguration: Configuration by configurations.creating {
+    isCanBeConsumed = false
+    isCanBeResolved = true
+}
+
+dependencies {
+    sharedConfiguration(project(path = ":producer", configuration = "sharedConfiguration"))
+}
+
+tasks.register("showFile") {
+    inputs.files(sharedConfiguration)
+    doFirst {
+        logger.lifecycle(sharedConfiguration.singleFile.absolutePath)
```

Comment:
Could you maybe check if this works with configuration cache? It would be good that all new samples work out of the box with it.

Reply:
The Kotlin wasn't compatible with the configuration cache. I've updated it. Thanks for asking me to check this.